.circleci/config.yml

```diff
@@ -3,33 +3,23 @@
 #
 version: 2
 jobs:
-  style-check:
+  repo-check:
     docker:
       - image: circleci/golang:1.13.0
 
-    working_directory: ~/cassandra-operator
+    working_directory: /go/src/github.com/instaclustr/cassandra-operator
     steps:
       - checkout
       - restore_cache: # restores saved cache if no changes are detected since last run
           keys:
             - go-mod-v4-{{ checksum "go.sum" }}
       - run:
-          name: Update go stuff
-          command: go get golang.org/x/tools/cmd/goimports && go mod download
-
+          name: Validate repository
+          command: GO111MODULE=on .circleci/validate_repo.sh
```

Comment:
isnt this GO111MODULE=on already default in 1.13?

Reply:
yes, but for some reason, it doesn't work without it being set.